manager/src/test/scala/org/broadinstitute/transporter/transfer/TransferSubmitterSpec.scala

```diff
@@ -296,8 +298,51 @@ class TransferSubmitterSpec extends PostgresSpec with MockFactory with EitherVal
             myTransfers should contain(id)
             submittedAt.isDefined shouldBe true
         }
-        submitted.foreach { _._1 should not be tId1 }
+        submitted.foreach {
+          _._1 should not be tId1
+        }
         totalSubmitted shouldBe submitted.length
       }
   }
+
+  it should "submit earlier eligible transfers of the same priority first" in withRequest {
+    val request3Id = UUID.randomUUID()
+    val ts = Timestamp.from(Instant.ofEpochMilli(0.toLong))
+    val request3Transfers = List.tabulate(3) { i =>
+      UUID.randomUUID() -> json"""{ "i": $i }"""
+    }
+
+    (tx, submitter) =>
+      (kafka.submit _)
+        .expects(where { (topic: String, submitted: List[TransferMessage[Json]]) =>
+          topic == theTopic &&
+          submitted.length == parallelism &&
+          submitted.forall { message =>
+            message.ids.request == request3Id &&
```

Comment:
Could you add a comment here calling out this line as the assertion that's checking we're only submitting the oldest-submitted transfers? It's easy to miss, so then it's tough to tell how this test is actually covering the new prioritization behavior.